### PR TITLE
Generate TypeScript strict mode compatible code.

### PR DESCRIPTION
**Pull request by ddunkin**

### CODE REVIEW

File: example/ts/exampleApi.ts [hunk 1]
```diff
@@ -366,7 +366,7 @@ class ExampleApiHttpClient implements IExampleApi {
 		return fetchResponse(this._fetch, this._baseUri + uri, fetchRequest)
 			.then(result => {
 				const status = result.response.status;
-				let value: IGetWidgetsResponse = null;
+				let value: IGetWidgetsResponse | null = null;
```
Comment (ejball) [inline example/ts/exampleApi.ts]: Can/should this be `IGetWidgetsResponse?` instead?

Comment (ddunkin) [reply example/ts/exampleApi.ts]: It cannot.

Comment (ejball) [reply example/ts/exampleApi.ts]: Why not?

Comment (ddunkin) [reply example/ts/exampleApi.ts]: Because it's not valid TypeScript. ;)

Comment (ejball) [reply example/ts/exampleApi.ts]: Oh! 😊

**Pull request merged**